spec/hako/scripts/create_log_group_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+require 'spec_helper'
+require 'hako/app_container'
+require 'hako/scripts/create_log_group'
+
+RSpec.describe Hako::Scripts::CreateLogGroup do
+  let(:script) { described_class.new(app, options, dry_run: false) }
+  let(:app) { double('Hako::Application', id: 'nanika') }
+  let(:options) do
+    {
+      'log_configuration' => {
+        'log_driver' => 'awslogs',
+        'options'    => {
```

Comment:
Sorry I was accustomed to align vertically.
Followed.

https://github.com/eagletmt/hako/pull/22/commits/8dba1c11a1b57d82e4537dc0b400da45a9b7f509